make.py

```diff
@@ -18,19 +18,27 @@ def create_fpga_graph():
     # Dependencies
     graph.add_input('acore_intf', ext='md', folders=['md'])
     graph.add_input('cdr_intf', ext='md', folders=['md'])
-    graph.add_input('sram_multi_intf', ext='md', folders=['md'])
-    graph.add_input('sm_ffe_intf', ext='md', folders=['md'])
     graph.add_input('dcore_intf', ext='md', folders=['md'])
+    graph.add_input('hist_intf', ext='md', folders=['md'])
```

Comment:
Not sure if this is really necessary, but I noticed that some Markdown files were missing from the build graph dependencies, so I added them.